tests/test_cdb.py

```diff
@@ -90,5 +90,6 @@ def test_cui2snames_population(self):
             with self.subTest(cui):
                 self.assertIn(cui, self.undertest.cui2snames)
 
+
```

Comment:
Small thing, but maybe remove this newline so it doesn't look like we changed this file.